### DISCUSSION TITLE
Updating without SIP disable on Mac

### DISCUSSION THREAD

**Discussion by ghost** [chris1111/Wireless-USB-Big-Sur-Adapter#35] [Ideas]
It is fabulous that you have developed the Big Sur driver for my Archer T4U wireless adaptor.  But, like nearly all Macs supplied by companies to their workers, they have hidden the safe boot behind a password.  The company IT support will not talk about allowing this to be changed.

What are the chances of this driver ever being able to be installed without SIP being disabled?

Peter

**Comment by chris1111** [comment]
I don't know Try it then let me known ?
If you not have a POPUP security allowing software extensions this is not work 
so you only have to select Uninstall to remove the program

**Comment by ghost** [comment]
If I try it without disabling SIP, the icon loads but the adaptor LED does not light, and the icon cannot be interacted with 😭
________________________________

**Comment by chris1111** [comment]
Thats why I mention to have the SIP deactivated
